CWE-94-Python_darwin/nccgroup_TriforceLinuxSyscallFuzzer/gen.py

```
@@ -0,0 +1,158 @@
+        x = Num(x)
+    x.mkArg(buf, xtra)
+
+def mkSyscall(nr, *args) :
+    args = list(args)
+    while len(args) < 6 :
+        args.append(0)
+
+    buf = Buf()
+    xtra = Buf()
+    buf.pack('!H', nr)
+    for n,arg in enumerate(args) :
```

Ruff (0.13.2)

Loop control variable `n` not used within loop body

Rename unused `n` to `_n`

(B007)